Test/comp/ForallNewSyntax.dfy

```diff
@@ -1,9 +1,9 @@
-// RUN: %dafny /quantifierSyntax:4 /compile:0 "%s" > "%t"
-// RUN: %dafny /quantifierSyntax:4 /noVerify /compile:4 /spillTargetCode:2 /compileTarget:cs "%s" >> "%t"
-// RUN: %dafny /quantifierSyntax:4 /noVerify /compile:4 /spillTargetCode:2 /compileTarget:js "%s" >> "%t"
-// RUN: %dafny /quantifierSyntax:4 /noVerify /compile:4 /spillTargetCode:2 /compileTarget:go "%s" >> "%t"
-// RUN: %dafny /quantifierSyntax:4 /noVerify /compile:4 /spillTargetCode:2 /compileTarget:java "%s" >> "%t"
-// RUN: %dafny /quantifierSyntax:4 /noVerify /compile:4 /spillTargetCode:2 /compileTarget:py "%s" >> "%t"
+// RUN: %baredafny verify %args --quantifier-syntax:4 "%s" > "%t"
+// RUN: %baredafny run %args --quantifier-syntax:4 --no-verify --target:cs "%s" >> "%t"
+// RUN: %baredafny run %args --quantifier-syntax:4 --no-verify --target:js "%s" >> "%t"
+// RUN: %baredafny run %args --quantifier-syntax:4 --no-verify --target:go "%s" >> "%t"
+// RUN: %baredafny run %args --quantifier-syntax:4 --no-verify --target:java "%s" >> "%t"
+// RUN: %baredafny run %args --quantifier-syntax:4 --no-verify --target:py "%s" >> "%t"
```

Comment:
Does `%baredafny %args run` work?  In tools like `git` you can pass common options between `git` and the command, e.g. `git --git-dir=… pull …`

Reply:
It might after we drop support for the old CLI UI, but right now we detect whether the first argument is a command to turn on the new UI

Reply:
!! I forgot.  Of course.